### DISCUSSION TITLE
Avoid multiple rapid hover state changes on chat reply link hover

### DISCUSSION THREAD

**Discussion by Moshyfawn** [zed-industries/zed#23923] [Feature Requests]
When hovering over the reply link in chat, the entire message container is first highlighted on hover, and then a moment later the reply link button gets its own hover state, resulting in a rapid UI change that's perceived as flickering. I would probably expect the second hover state to change only the cursor type rather than the whole background.

https://github.com/zed-industries/zed/assets/16290753/588aebbc-8c12-412b-a277-41e7296a6e92

**Comment by maxdeviant** [comment]
I agree, I think we could present this better visually. Going to move to a discussion until we revisit chat.
